morphia/src/main/java/org/mongodb/morphia/Datastore.java

```diff
@@ -575,16 +575,30 @@
      * @param <T>      the type of the entity
      * @param wc       the WriteConcern to use
      * @return the keys of the entities
+     * @deprecated use {@link #save(Iterable, InsertOptions)} instead
      */
+    @Deprecated
     <T> Iterable<Key<T>> save(Iterable<T> entities, WriteConcern wc);
 
     /**
+     * Saves the entities (Objects) and updates the @Id field, with the WriteConcern
```

Comment:
Why does this specifically call out the update to the @Id field?   Does this save method have different behavior than the one in Datastore?  I would expect all save, and insert methods, to update the @Id-annotated field.  

Let's use consistent language regarding update to @Id.  Either call it out everywhere it happens, or nowhere.